lib/alephant/broker/response/base.rb

```diff
@@ -19,20 +19,20 @@ class Base
           NOT_MODIFIED_STATUS_CODE => "",
           404                      => "Not found",
           500                      => "Error retrieving content"
-        }
+        }.freeze
```

Comment:
shame constants aren't immutable in Ruby as putting `freeze` seems so unnecessary otherwise (due to the understanding that constants should be just that... constant)